classes/hook_callbacks.php

```diff
@@ -53,4 +53,128 @@ public static function before_footer_html_generation(\core\hook\output\before_fo
 
         tool_abconfig_execute_js('footer');
     }
+
+    /**
+     * Runs after config has been set.
+     *
+     * @param \core\hook\before_http_headers $hook
+     * @return void|null
+     */
+    public static function after_config(\core\hook\after_config $hook) {
+        if (!get_config('tool_abconfig', 'version')) {
```

Comment:
it looks like you copied the code from `tool_abconfig_after_config`. Is there any way to abstract  the to avoid code duplication?

Reply:
That's right - it is just copy and pasted. As we're not creating separate stable branches and the function of the callback remains the same, we could just call the original `tool_abconfig_after_config` function from the new hook.

Reply:
Hi @djarran ,

Please update the PR with the call to existing tool_abconfig_after_config, I will review and merge the PR

Reply:
Hi @tuanngocnguyen, this has been done now. Thank you